EIPS/eip-1.md

```diff
@@ -19,29 +20,59 @@ For Ethereum implementers, EIPs are a convenient way to track the progress of th
 
 ## EIP Types
 
-There are three types of EIP:
+There are three types of EIP: Standards Track, Meta and Informational.
```

Comment:
It might. It might also be worth pulling all of the EIPs together into one working group-- there aren't that many Networking and Interface proposals, they tend to be more closely related to Core work than ERCs, (some require client changes?) and the core devs are probably more competent than most to review them.

Reply:
If we do that, Victor and I have considered creating an ERC working group, if he can find a few other people to help.  If we can work out how to set up repos for each group something close to a split could be realized without an actual bifurcation.

Reply:
I would be happy to join an ERC WG.